### ISSUE TITLE
emscripten build fails with CMakeError (include could not find load file UseEmscripten)

### ISSUE THREAD

**Issue by DerThorsten** [mosra/corrade#104]
without #103   the emscripten build fails for me with:
```
CMake Error at modules/UseCorrade.cmake:319 (include):
  include could not find load file:

    UseEmscripten
Call Stack (most recent call first):
  CMakeLists.txt:225 (include)


CMake Error at CMakeLists.txt:227 (include):
  include could not find load file:

    UseEmscripten


-- Configuring incomplete, errors occurred!
```

**Comment by mosra** [comment]
Hi!

You should have asked first, would save you an unnecessary PR ;) I suppose this is because you're not using Corrade's own toolchain file for Emscripten [as described here](https://doc.magnum.graphics/corrade/building-corrade.html#building-corrade-cross-emscripten)? That one sets up the module path for `UseEmscripten` correctly, the other directory [is already added there](https://github.com/mosra/corrade/blob/d0392ea557439862537bcda35119d7d45c21034d/CMakeLists.txt#L158).

That being said, the `UseEmscripten` module shouldn't be strictly necessary -- Corrade itself doesn't use it except for some tests, which aren't built by default anyway. I'll patch the code to avoid this extra dependency, which should also allow 3rd party toolchains to be used.

**Comment by mosra** [comment]
This should be fixed with 9bc39504af48f5645b326bcbf37a3187bbbcbfa6 -- `UseEmscripten` is no longer implicitly needed and thus it's possible to build Corrade with 3rd party toolchain files as well.
